docs/alt-installation.md

```diff
@@ -50,3 +50,22 @@ github.com/earthly/hello-world:main+hello | Hello, world!
 github.com/earthly/hello-world:main+hello | Target github.com/earthly/hello-world:main+hello built successfully
 =========================== SUCCESS ===========================
 ```
+
+# Uninstall
+
+To remove earthly, run the following commands:
+
+    # macOS users
+    brew uninstall earthly
+    rm -rf ~/.earthly
+    docker rm --force earthly-buildkitd
+    docker volume rm --force earthly-cache
+
+    # Linux and WSL2 users
```

Comment:
Can you make these proper subsections, with their own code boxes? It helps the user better visually separate the instructions.

Reply:
d'oh! I don't know what I'm thinking here with this. Let me fix that.